JS/Lesson32/savas/my-app/src/components/RecipeDetail/index.jsx

```diff
@@ -0,0 +1,31 @@
+import React from 'react';
+
+export const RecipeDetail = ({recipe}) => {
+  return (
+    
+    <div>
+        <h2>{recipe.strMeal}</h2>
+        <h3>Country: {recipe.strArea}</h3>
+        <a href={recipe.strSource}>Original Recipe Link</a>
+        <h3>Ingredients:</h3>
+        <ul>
+            <li>{recipe.strIngredient1}</li>
```

Comment:
You have to repeat yourself a lot when creating `li` elements manually. Instead, try to automate it. Every recipe has `strIngredient` with indexes from 1 to 20 so you'd need to write some JS code that would get all of this 20 values and then display them here in the component. 
It would help a lot if you try to get these 20 values and put the in an array variable first. Then, inside  `ul`, just map through the array and display `li` for each element in array.